src/export/InstanceExporter.ts

```diff
@@ -92,9 +92,13 @@ export class InstanceExporter implements Fishable {
           if (foundFixedValue) {
             // Find how much the two paths overlap, for example, a.b.c, and a.b.d overlap for a.b
             let overlapIndex = 0;
-            element.id.split('.').forEach((p, i) => {
-              if (p === associatedEl.id.split('.')[i]) overlapIndex++;
-            });
+            for (const [i, p] of element.id.split('.').entries()) {
+              if (p === associatedEl.id.split('.')[i]) {
+                overlapIndex++;
+              } else {
+                break;
+              }
+            }
```

Comment:
Probably not a big deal, but you're splitting `associateEl.id` on every iteration.  It would be more efficient to pull it out.  Also, noticing that I think `i` and `overlapIndex` are pretty closely related -- and the `break` can happen automatically in a traditional for loop.  Is this too fancy?

```ts
const elParts = element.id.split('.');
const assocElParts = associatedEl.id.split('.');
let overlapIdx = 0;
for (; overlapIdx < elParts.length && elParts[overlapIdx] === assocElParts[overlapIdx]; overlapIdx++);
```

Reply:
Not too fancy, in fact I'd say just fancy enough. So it's added in https://github.com/FHIR/sushi/pull/358/commits/a5b68fd8aba2a34bc822abe127daa6f624a371bf.